resources/views/components/nav.blade.php

```diff
@@ -0,0 +1,51 @@
+@props(['nav' => 'nav:main', 'mobileNav' => 'nav:main'])
+@php
+    // This combines multiple navigations into one for desktop and mobile seperately, only neccessary if you have multiple navigations
+    $navData = array_merge(...Arr::map(Arr::wrap($nav), fn($item) => StatamicNav::nav($item)));
```

Comment:
These are not too readable, i suggest putting it in a Collection instead and using those functions.

Reply:
Agreed